src/Visualization/Python/PlotCce.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+from typing import Optional, Sequence
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from spectre.Visualization.Plot import (
+    apply_stylesheet_command,
+    show_or_save_plot_command,
+)
+from spectre.Visualization.ReadH5 import available_subfiles, to_dataframe
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_modes(ctx, param, all_modes):
+    result = []
+    for mode in all_modes:
+        l_and_m = mode.split(",")
+        if len(l_and_m) != 2:
+            raise click.BadParameter(
+                f"The mode {mode} must be specified as 'l,m'"
+            )
+        result.append(f"Real Y_{mode}")
+        result.append(f"Imag Y_{mode}")
+    return result
+
+
+@click.command(name="cce")
+@click.argument(
+    "h5_filename",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+    nargs=1,
+)
+@click.option(
+    "--modes",
+    "-m",
+    multiple=True,
+    callback=_parse_modes,
+    required=True,
+    help=(
+        "Which mode to plot. Specified as 'l,m'. Will plot both real and"
+        " imaginary components unless '--real' or '--imag' are specified. Can"
+        " be specified multiple times."
+    ),
+)
+@click.option(
+    "--real",
+    is_flag=True,
+    default=False,
+    help="Plot only real modes. Mutually exclusive with '--imag'.",
+)
+@click.option(
+    "--imag",
+    is_flag=True,
+    default=False,
+    help="Plot only imaginary modes. Mutually exclusive with '--real'.",
+)
+@click.option(
+    "--extraction-radius",
+    "-r",
+    type=int,
+    help=(
+        "Extraction radius of data to plot as an int. If there is only one Cce"
+        " subfile, that one will be used and this option will be ignored if"
```

Comment:
Don't silently ignore it. Use an XOR in the check below where you raise a usage error, or assert that that the specified radius matches the single available subfile name.

Reply:
This seems a bit redundant especially since this is only meant to give a quick plot. If I have to specify this when there is only one subfile, then I have to `h5ls` the H5 file (or run this with `-l`) first and then copy the extraction radius and (re)run the command.

Reply:
No I mean only if it _is_ specified, assert that the value is consistent with the single existing subfile. Also allow not specifying it if there's only one subfile.

Reply:
Oh my bad, I misunderstood. Yes I agree this would be good